hooks/useAcademcInfo.tsx

```diff
@@ -25,6 +26,7 @@ function useAcademicInfo() {
   };
 
   useEffect(() => {
+    if (inputRef.current) inputRef.current.focus();
```

Comment:
This logic seems to be more appropriate to be placed in the view component, as the hook is growing up in side effects and responsibilities